src/main/java/life/qbic/expdesign/io/MHCLigandDesignReader.java

```diff
@@ -138,7 +139,32 @@ public MHCLigandDesignReader() {
     headersToTypeCodePerSampletype.put(SampleType.Q_MHC_LIGAND_EXTRACT, ligandsMetadata);
     // headersToTypeCodePerSampletype.put("Q_MS_RUN", msRunMetadata);
   }
+  private void fillParsedCategoriesToValuesForRow(Map<String, Integer> headerMapping,
+      String[] row) {
+    // logger.info("Collecting possible CV entries for row.");
+    addValueForCategory(headerMapping, row, "Organism");
+    addValueForCategory(headerMapping, row, "Tissue");
+    addValueForCategory(headerMapping, row, "Dignity");
+    addValueForCategory(headerMapping, row, "Tumor Type");
+    addValueForCategory(headerMapping, row, "TNM");
+    addValueForCategory(headerMapping, row, "Antibody");
+    addValueForCategory(headerMapping, row, "MHC Class");
```

Comment:
Down the line you could think about moving these i assume default values into a seperate enum defaultRowHeaders. 
(The Same could be done for the default map values put and assigned to MHCClass in the
antibodyToMHCClass Map)